be/src/exec/broker_scan_node.cpp

```diff
@@ -407,7 +407,6 @@ void BrokerScanNode::scanner_worker(int start_idx, int length) {
     }
 
     // Update stats
-    // _runtime_state->update_num_rows_load_success(counter.num_rows_returned);
```

Comment:
You should remove `update_num_rows_load_success ` in RuntimeState, and its corresponding variables